package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "algoliasearch-extensions-bundle",
-  "version": "4.3.9",
+  "version": "4.4.0-v3",
```

Comment:
this isn't valid semver, what do you want v3 to mean? Valid version would be `4.4.0-v3.0`

Reply:
This is just a suffix for extension identification. Not meant for versioning.